frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -249,17 +239,32 @@ const TestCardForm = ({
   };
 
   const validateTestResults = () => {
```

Comment:
Any chance we could rename these and the related `validate....` functions to something like `generateTestResultsErrors`? It felt weird to me that we were returning an empty string in a validation function like this (would have expected a boolean / undefined / null return in the case of this is / isn't valid) but see instead that we're using these to generate an error state, which makes me think that these functions are misnamed. 

Along the same lines, I think it might be more readable to return something like `null` here and then do an explicit null check on 271 and related places. That would divorce the content of the error from the state of whether the error exists or not which aren't the same. The check would read something like
```
const testResultErrors = generateTestResultErrors()
const showTestResultsError = hasAttemptedSubmit && testResultsError != null;
....
   if (testResultsError) { // or if(testResultsError !== null)
      showError(testResultsError, "Invalid test results");
    }
```

Reply:
Yeah I went back and forth on whether the name should mention validation. Does `getTestResultsError` and `getDateTestedError` sound good? I'll also update those functions to return `null` instead of empty string